sale_order_warn_message/models/sale_order.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2020 ForgeFlow S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class SaleOrder(models.Model):
+
+    _inherit = "sale.order"
+
+    sale_warn_msg = fields.Text(compute="_compute_sale_warn_msg")
+
+    def _compute_sale_warn_msg(self):
+        for rec in self:
+            if rec.partner_id and rec.state in ["draft", "sent"]:
+                if (
+                    rec.partner_id.parent_id
+                    and rec.partner_id.parent_id.sale_warn == "warning"
+                ):
+                    rec.sale_warn_msg = rec.partner_id.parent_id.sale_warn_msg
+                    if rec.partner_id.sale_warn == "warning":
+                        rec.sale_warn_msg += "\n%s" % rec.partner_id.sale_warn_msg
+                        continue
+                elif rec.partner_id.sale_warn == "warning":
+                    rec.sale_warn_msg = rec.partner_id.sale_warn_msg
+                    continue
```

Comment:
why don't use 'commercial_partner_id' field?
Why do you use 'continue'?  Better to use a variable res and write this value to the register at the end. ¿no?

Reply:
I know it should use commercial_partner_id, it would reduce conditionals, but this module is based in https://github.com/OCA/account-invoicing/pull/698, and I would like them to be alike. Thus, first convince @HviorForgeFlow to use that field, and then I will apply it here :)

Reply:
I'm convinced, do the change :clinking_glasses:

Reply:
done